com.unity.shadergraph/Editor/Generation/Utils/TargetUtils.cs

```diff
@@ -3,6 +3,7 @@
 using System.Collections.Generic;
 using UnityEditor.Graphing;
 using UnityEditor.ShaderGraph.Serialization;
+using UnityEngine.Pool;
```

Comment:
The use of ListPool in this file doesn't seem like it would give us any advantage over just allocating a List<>

Do we want to remove the use of a pool here?